### PR TITLE
Shell script to build v1/original exome calling intervals (with 150bp padding)

### PR DESCRIPTION
**Pull request by ldgauthier**

### CODE REVIEW

File: scripts/generate_hg38_exome_calling_intervals.v2.sh [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+# =============== mirbase.v22.interval_list ================
+# ==========================================================
+
+if [ ! -e mirbase.v22.gff ]; then 
+wget -O mirbase.v22.gff ftp://mirbase.org/pub/mirbase/22/genomes/hsa.gff3
+fi
+
+cp $hg38/Homo_sapiens_assembly38.dict tmp.interval_list
+cat mirbase.v22.gff | awk '$3 == "miRNA_primary_transcript" {sub(/.*Name=/, "", $9); print $1,$4,$5,$7,$9}' | tr ' ' '\t' >> tmp.interval_list
+java -jar $picard IntervalListTools \
```
Comment (kcibul) [inline scripts/generate_hg38_exome_calling_intervals.v2.sh:38]: this command complains for me  

```
java.lang.IllegalStateException: Interval list file must contain header.
```

**Comment by gbggrant** [comment]
Hi @ldgauthier do you want to move ahead with this PR?  I'm not sure why it has a 'DO NOT MERGE' tag on it. @kbergin ?

**Comment by kbergin** [comment]
We had considered trying to deploy this with the joint calling reblocking updates but decided not to, I'm going to guess Laura isn't eager to pick it back up right now because she has other exciting things to work on. @ldgauthier perhaps we can chat soon to decide if this is a "later" or a "never".

**Comment by kbergin** [comment]
Haven't gotten to talk to Laura about this yet.

**Comment by ldgauthier** [comment]
IIRC the blocker to implementing/merging this was being able to track GVCF "versions" in CLIO or somewhere, right?  Or does it matter if we also reduce the padding on the JG intervals?  I honestly can't remember where we left this.

**Comment by ldgauthier** [comment]
This is the script that goes with the exome intervals we are *currently* using, so effectively adding provenance for what we have.  Shouldn't affect any tests or anything.

**Comment by jessicaway** [comment]
@ldgauthier Are we still interested in getting this merged?

**Comment by ldgauthier** [comment]
Yes please.  It's for provenance and it won't change any pipelines so it should be just a rubber stamp approval.

**Review by jessicaway — APPROVED** [review]

**Comment by jessicaway** [comment]
test this please

**Review by nikellepetrillo — APPROVED** [review]

**Review by khajoue2 — APPROVED** [review]

**Comment by jessicaway** [comment]
test this please

**Pull request merged**